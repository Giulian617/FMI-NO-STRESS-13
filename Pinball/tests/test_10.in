10 38
5 7 8
2 5 2
9 2 4
10 9 8
4 10 7
3 4 2
1 3 6
6 1 10
8 6 8
