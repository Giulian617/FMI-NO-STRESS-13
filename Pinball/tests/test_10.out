375000003
625000005
